dxvk/dxvk.go

```diff
@@ -36,6 +36,12 @@ func Remove(pfx *wine.Prefix) error {
 
 			slog.Info("Removing DXVK overriden Wine DLL", "path", p)
 
+                        // Checks to see if the given file exists, if not, skips it and continues removal process
+                        if _, err := os.Stat(p); os.IsNotExist(err) {
+                                slog.Info("File does not exist, skipping", "path", p)
+                                continue
+                        }
```

Comment:
This error checking is not reccomended:
> New code should use errors.Is(err, fs.ErrNotExist).

This error checking should be moved to only check the error of the removal of the DLL, which should always be present. It not being present means that the user has manipulated with the wineprefix and Vinegar can no longer determine if it is installed or not.